pkgs/development/web/deno/default.nix

```diff
@@ -12,23 +13,18 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "deno";
-  version = "1.33.3";
+  version = "1.34.0";
 
   src = fetchFromGitHub {
     owner = "denoland";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-E1Usmj4f3jvzllKIrItzOIe4kf0DRQWwobVGuN291Ys=";
+    hash = "sha256-B9YzJsnbohzQQQj2Ec1szh/aY2zoYFOoEXTShwOzMlE=";
   };
-  cargoHash = "sha256-8L9UrFQTpwG2Gw4V72NvqdqQicG0HPRdmNJPhkWqj4s=";
+  cargoHash = "sha256-rwN7pEb6bDLb7YrcfuWC9Y0m7b+50AeLKMKx7WLKYT8=";
 
-  postPatch = ''
-    # upstream uses lld on aarch64-darwin for faster builds
-    # within nix lld looks for CoreFoundation rather than CoreFoundation.tbd and fails
-    substituteInPlace .cargo/config.toml --replace '"-C", "link-arg=-fuse-ld=lld"' ""
-  '';
-
-  nativeBuildInputs = [ installShellFiles ];
+  nativeBuildInputs = [ installShellFiles ] ++
+    lib.optionals (stdenv.isDarwin && stdenv.isAarch64) [ rustc.llvmPackages.lld ];
```

Comment:
Looks like it's breaking the aarch64-darwin build? #234135 doesn't have the issue.

Reply:
I try to use lld like upstream every so often to see if its working within nixpkgs. I use ofborg to test it as I dont have an aarch64 machine.

This PR does drop TCC though which r-ryantm update doesnt